src/services/opensrp/index.ts

```diff
@@ -72,7 +72,7 @@ export class OpenSRPService extends OpenSRPServiceWeb {
   }
 
   public async readFile(
-    id: string | number,
+    id?: string | number,
```

Comment:
Why is this one necessary?

Reply:
The jurisdictions settings endpoint doesn't need an id URL param so i had to make the id param optional

Reply:
cool